service/src/main/java/org/sunbird/service/otp/OTPService.java

```diff
@@ -38,10 +42,28 @@ public void deleteOtp(String type, String key, RequestContext context) {
     otpDao.deleteOtp(type, key, context);
   }
 
-  public static String getSmsBody(
+  /**
```

Comment:
Can we call the user service directly in otp actor and decryption be done in user service itself?